WordPressComKit/RequestRouter.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import Alamofire
+
+public enum RequestRouter: URLRequestConvertible {
+    static let baseURLString = "https://public-api.wordpress.com/rest/v1.1/"
```

Comment:
I am totally against hitting different versions of the API with a shared library of model objects based off of a single version. This is a shortcoming of how we handle versioning with WordPress.com's API of course. I'm really apprehensive about supporting the hack we have to support in the main WPiOS app but I'll try and come up with a way to gracefully override.